src/wp-includes/class-wp-block-bindings-source.php

```diff
@@ -74,15 +73,31 @@ public function __construct( string $name, array $source_properties ) {
 	 * Retrieves the value from the source.
 	 *
 	 * @since 6.5.0
+	 * @since 6.7.0 `block_bindings_source_value` filter was added.
 	 *
-	 * @param array    $source_args     Array containing source arguments used to look up the override value, i.e. {"key": "foo"}.
-	 * @param WP_Block $block_instance  The block instance.
-	 * @param string   $attribute_name  The name of the target attribute.
+	 * This function calls the callback function specified in the `$get_value_callback` property
+	 * with the given arguments and returns the result. It then applies the filter
+	 * `block_bindings_source_value` to the value before returning it.
 	 *
-	 * @return mixed The value of the source.
+	 * @param array    $source_args    Array containing source arguments used to look up the override value, i.e. {"key": "foo"}.
+	 * @param WP_Block $block_instance The block instance.
+	 * @param string   $attribute_name The name of the target attribute.
+	 * @return mixed                   The value of the source.
```

Comment:
Nit: Does it really need to have extra spaces? I don't think it's used like this in other places.
```suggestion
	 * @return mixed The value of the source.
```

Reply:
Just one space.